cloud/organization/organization.go

```diff
@@ -184,6 +184,7 @@ func ExportAuditLogs(client astro.Client, out io.Writer, orgName string, earlies
 		return err
 	}
 	logStreamBuffer.Close()
+	fmt.Println("Done")
```

Comment:
Can we also add the path to this print message to say where the logs are being exported to?